drivers/soundwire/generic_bandwidth_allocation.c

```diff
@@ -190,6 +190,21 @@ static int sdw_compute_group_params(struct sdw_bus *bus,
 	}
 
 	list_for_each_entry(m_rt, &bus->m_rt_list, bus_node) {
+		/*
+		 * sdw_compute_group_params could be called in the deprepared case, skip the
+		 * m_rt in that case.
+		 */
+		if (m_rt->stream == stream)
+			if (stream->state != SDW_STREAM_CONFIGURED)
+				continue;
+		/*
+		 * Stream stage ENABLED and DISABLED are between PREPARED and DEPREPARED,
+		 * We should count the bandwidth only for ENABLED and DISABLED stages.
```

Comment:
@bardliao I think looking at the enum definition, its easy to see this. A better explanation would be something like "Include runtimes with running (ENABLED state) and paused (DISABLED state) streams"

Reply:
Both updated.